sudoers/files/sudoers

```diff
@@ -95,6 +98,13 @@ Runas_Alias {{ name }} = {{ ",".join(runas) }}
   {%- endfor %}
 {%- endfor %}
 
+# Network Group privilege specification
+{%- for group,specs in network_groups.items() %}
```

Comment:
Is the plus sign needed for the logic here on lines 102, 103, 105, and 106? Should we just have `+` signs before and after the logic?